x11-plugins/purple-hangouts/purple-hangouts-0_p20161222.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Hangouts Plugin for libpurple"
+HOMEPAGE="https://bitbucket.org/EionRobb/purple-hangouts"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit mercurial
+	EHG_REPO_URI="https://bitbucket.org/EionRobb/purple-hangouts"
+else
+	COMMIT_ID="754e3bb971cf"
+	SRC_URI="https://bitbucket.org/EionRobb/purple-hangouts/get/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/EionRobb-${PN}-${COMMIT_ID}"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
+	dev-libs/glib:2=
+	dev-libs/json-glib
+	dev-libs/protobuf-c:=
+	net-im/pidgin"
+
+DEPEND="${RDEPEND}
```

Comment:
According to ``purple-hangouts.spec`` this package also has a build-time-only dependency on zlib and this appears to be confirmed by ``purple2compat/http.c``.